.devcontainer/Dockerfile

```diff
@@ -1,24 +1,13 @@
-ARG ELIXIR_VERSION
-FROM elixir:${ELIXIR_VERSION}
+ARG IMG
+FROM ${IMG}
```

Comment:
Maybe change the variable name to something more descriptive, like 'COUCHDB_IMAGE' or whatever you like.

Reply:
Will do!